Activity messages now fill all the horizontal space available

msfernandes opened the pull request:
Fixes: #487 

Large screen:
![screenshot-localhost 5000 2016-03-06 01-37-23](https://cloud.githubusercontent.com/assets/2891076/13552339/5de32a1e-e33d-11e5-87a2-d7a0a6e40a7b.png)

Medium screen:
![screenshot-localhost 5000 2016-03-06 01-38-13](https://cloud.githubusercontent.com/assets/2891076/13552341/5fb06ffa-e33d-11e5-9af9-be8367a939fa.png)

Small screen:
![screenshot-localhost 5000 2016-03-06 01-38-41](https://cloud.githubusercontent.com/assets/2891076/13552342/6168504c-e33d-11e5-9cec-21408ef2a2f6.png)

When the namespace, for example, is too large, the div with the description exceeds the Recent Activities div:
![screenshot-localhost 5000 2016-03-06 01-58-43](https://cloud.githubusercontent.com/assets/2891076/13552379/003e2c40-e33f-11e5-80df-b060227fbfc3.png)

So I added `overflow: hidden` for avoid this layout break (see small screenshot). But I think that is better set a max length to these names.

Signed-off-by: Matheus Fernandes <matheus.souza.fernandes@gmail.com>

cyphar commented:
What about if we wrap it, I'm not sure about truncating messages if they're long. I'd prefer making them taller by wrapping the text.

msfernandes commented:
`text-overflow` property maybe solve this problem, truncating text. As soon as possible I'll fix it.

msfernandes commented:
I could not do using html and css, so I used the `truncate` helper from Rails to truncate namespaces, team names, etc, at 40 characters length. But it truncates in large screen too. 

![screenshot-localhost 5000 2016-03-07 10-26-26](https://cloud.githubusercontent.com/assets/2891076/13570783/301bc2b8-e44f-11e5-85f1-172fdf4c4068.png)

![screenshot-localhost 5000 2016-03-07 10-26-58](https://cloud.githubusercontent.com/assets/2891076/13570782/2ef62180-e44f-11e5-9f25-1445ed67a373.png)

This length is ok?

mssola commented:
Not an expert by any means when it comes to CSS, but wouldn't [this](https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/) work ? If that's not the case, maybe truncating into 40 chars is not too bad. We can fix that later.

msfernandes commented:
Removed rails truncate and added trucante on css!
![screenshot-localhost 5000 2016-03-07 13-22-23](https://cloud.githubusercontent.com/assets/2891076/13575614/923e2a62-e468-11e5-880e-aaea4c13310c.png)
![screenshot-localhost 5000 2016-03-07 13-27-18](https://cloud.githubusercontent.com/assets/2891076/13575617/948ecd94-e468-11e5-9c2d-92751c23870e.png)
![screenshot-localhost 5000 2016-03-07 13-21-40](https://cloud.githubusercontent.com/assets/2891076/13575612/913a86f6-e468-11e5-93f1-99183585e895.png)

mssola commented:
Thanks :clap:
